modules/SpyUtils.js

```diff
@@ -4,6 +4,10 @@ import { isFunction } from './TestUtils'
 
 const noop = () => {}
 
+const engineSupportsConfigurableFunctionLength = (function () {
+  return Object.getOwnPropertyDescriptor(() => {}, 'length').configurable
```

Comment:
alternatively, add a dependency and use https://github.com/ljharb/define-properties/blob/master/index.js#L54